### ISSUE TITLE
RetinaNet: How to create customized Accuracy metrics and IOU metrics function for model.fit

### ISSUE THREAD

**Issue by mj1156** [keras-team/keras-io#226]
I am trying to code for customized functions for accuracy and IOU using the link :
https://keras.io/api/metrics/#custom-metrics but I am getting errors while running it .

**Comment by srihari-humbarwadi** [comment]
Please use [pycocotools/cocoeval.py](https://github.com/cocodataset/cocoapi/blob/master/PythonAPI/pycocotools/cocoeval.py) for evaluating the model. You can wrap this functionality inside a custom callback that runs after each epoch. More about custom callbacks [here](https://keras.io/api/callbacks/)

**Comment by myopengit** [comment]
Any example of how to calculate the accuracy metric?  I have used the default model in https://github.com/keras-team/keras-io/blob/master/examples/vision/retinanet.py to generate a json file and use the cocoeval.py to test it. But it seems the accuracy is just 0.044. Any examples will be greatly appreciated.

**Comment by 05vald0** [comment]
Any updates on this topic? I am also trying to implement an IOU metric for model.fit and don't see how can I use cocoeval.py, since I don't have a json file with annotations. An example of this would be great.

**Comment by pcoet** [comment]
Hi folks, it seems like this is a support question. If it's still a relevant question, please raise it in the forums below:

* [TensorFlow forum](https://discuss.tensorflow.org/).
* [Keras mailing list](https://groups.google.com/forum/#!forum/keras-users).

Thanks!
